configdb/hardware/models.py

```diff
@@ -173,8 +188,10 @@ def __str__(self):
 
 
 class ConfigurationTypeProperties(BaseModel):
```

Comment:
should add a verbose_name_plural to this one as well, it's current plural has two "s" at the end